images/linux/scripts/helpers/github-api.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+################################################################################
+##  File:  github-api.sh
+##  Desc:  Helper functions for github api
+################################################################################
+
+function getLatestReleaseURI {
+     if [[ $# -eq 0 || "x$1" = "x-h" || "x$1" = "x--help" ]];then
+        echo "usage: " >&2
+        echo "       $0 organisation/repo format" >&2
+        echo "       $0 organisation repo format" >&2
+        echo >&2
+        echo " Format should be .deb, .tar.gz or other download file extension" >&2
+        exit 1
+    fi
+
+    if [[ "x$1" == *"/"* ]]; then
+        url=$1
+        format=$2
+    else
+        url="$1/$2"
+        format=$3
+    fi
+
+    jq_script=".assets[].browser_download_url|select(contains(\"linux\") and contains(\"amd64\") and contains(\"${format}\"))"
```

Comment:
I like approach with helper functions but considering this condition it is very specific and can't be re-used anywhere.
Such conditions work for GH CLI but could be very different for other tools

Reply:
Nope. It works for any github release for ubuntu

Reply:
What about 
- `https://api.github.com/repos/julialang/julia/releases/latest`
- `https://api.github.com/repos/SeleniumHQ/selenium/releases/latest`
- `https://api.github.com/repos/mozilla/geckodriver/releases/latest`
- `https://api.github.com/repos/docker/composereleases/latest`

These conditions will work only in case if release asset naming contains linux / amd64 works but it is special case when asset name satisfy this template. The most of the releases have own naming and this function won't work for them.
So this helper is not generic one because it works correctly only with this specific repo

Reply:
Ah, i planned to add more or less generalised  helpers functions on demand but that would be out of scope of this PR. 

What about renaming the function to getLatestLinuxAMDReleaseURI and let it alone till we need to handle other formats

Reply:
I don't really think that we need this helper function because if we remove help and processing of different input parameters, function can be reduced to 2 lines:
```
jq_script=".assets[].browser_download_url|select(contains(\"linux\") and contains(\"amd64\") and contains(\"${format}\"))"
curl -s https://api.github.com/repos/${url}/releases/latest | jq -r "$jq_script"
```
Considering the fact that the first line will different for any repo, function won't be re-usable.

I like the direction of reworking common logic to helper functions but in this particular case, it is not needed because process of getting download link from releases if pretty specific and can't generalized